app/common/state/tabContentState/titleState.js

```diff
@@ -2,24 +2,64 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+const locale = require('../../../../js/l10n')
+
 // State helpers
+const partitionState = require('../tabContentState/partitionState')
+const privateState = require('../tabContentState/privateState')
 const frameStateUtil = require('../../../../js/state/frameStateUtil')
-const locale = require('../../../../js/l10n')
+
+// Utils
+const {isEntryIntersected} = require('../../../../app/renderer/lib/observerUtil')
+
+// Styles
+const {intersection} = require('../../../renderer/components/styles/global')
+
+module.exports.showTabTitle = (state, frameKey) => {
+  const frame = frameStateUtil.getFrameByKey(state, frameKey)
+
+  if (frame == null) {
+    if (process.env.NODE_ENV !== 'test') {
+      console.error('Unable to find frame for showTabTite method')
```

Comment:
s/showTabTite/showTabTitle

Reply:
wow eagle eyes. I left a https://github.com/brave/browser-laptop/pull/10691#discussion_r139507556 and TL;DR will eventually remove all warnings